src/main/java/com/google/sps/servlets/schedule/ScheduleGenerationServlet.java

```diff
@@ -78,7 +79,67 @@ protected void doGet(HttpServletRequest request, HttpServletResponse response)
     String id = (String) calendar.get("id");
     request.getSession().setAttribute("study-schedule-id", id);
 
-    httpClient.getConnectionManager().shutdown();
+    ////////////////////////// 2) Create Events on Calendar
+
+    List<String> resources = (List<String>) request.getSession(false).getAttribute("resources"); // TODO: get from data store
+    System.out.println(resources);
+
+    // Grab timezone to be used with creation of event.
+    JSONObject timeZoneSetting = getSetting(accessToken, "timezone");
+    String timezone = (String) timeZoneSetting.get("value");
+    TIME.setTimeZoneId(timezone);
+
+    // Let's try to create events! We depend on RECURRING EVENTS. So, upto the next week, we try to create 
+    Integer successfulEventsCreated = 0;
+
+    // Set user calendars to get FreeBusy info here
+    List<String> ids = getAllCalendarIds(httpClient, accessToken);
+
+    Integer stepValue = USER.EVENT_LOOK_SPAN / 2;
```

Comment:
Why not just 'int'?

Java is able to convert between the two easily and the raw type int is usually preferred

Reply:
Right!